src/Module/Api/Twitter/DirectMessages/NewDM.php

```diff
@@ -0,0 +1,77 @@
+<?php
+/**
+ * @copyright Copyright (C) 2010-2021, the Friendica project
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+
+namespace Friendica\Module\Api\Twitter\DirectMessages;
+
+use Friendica\Database\DBA;
+use Friendica\DI;
+use Friendica\Model\Contact;
+use Friendica\Model\Mail;
+use Friendica\Module\BaseApi;
+use Friendica\Network\HTTPException\NotFoundException;
+
+/**
+ * Sends a new direct message.
+ *
+ * @see https://developer.twitter.com/en/docs/direct-messages/sending-and-receiving/api-reference/new-message
+ */
+class NewDM extends BaseApi
```

Comment:
Please rename this class to `New` since we're already in the `DirectMessage` namespace. And it would keep the match with the endpoint path as well.

Reply:
`New` cannot be used as a class name.

Reply:
Reserved word, huh, never mind then.